spec/stats_integration_spec.rb

```diff
@@ -0,0 +1,34 @@
+require_relative 'spec_helper'
```

Comment:
test could use a better name - it's a unit test really and integration is a rather overloaded term